### PR TITLE
feat: ledger polkadot generic app

### PR DESCRIPTION
**Pull request by 0xKheops**

### CODE REVIEW

File: apps/playground/src/components/Substrate/balances/SendNative.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        const unsub = await api.tx.balances
+          .transferAllowDeath("5EXb7e8Kq9m62XTFKVYmGsHFADU4knyFNg6NKJmLKDCz4Gij", data.plancks)
+          .signAndSend(account.address, (result) => {
+            const { status } = result
+            setStatus(status)
+
+            if (status.isFinalized) {
+              unsub()
+              setTxProcessing(false)
+              // refresh()
```
Comment (chidg) [inline apps/playground/src/components/Substrate/balances/SendNative.tsx]: needed?

Comment (0xKheops) [reply apps/playground/src/components/Substrate/balances/SendNative.tsx]: definitely not, left over from some other component's copy paste.
thanks

File: packages/chaindata-provider/src/types/Chain.ts [hunk 2]
```diff
@@ -39,6 +39,8 @@ export type Chain = {
   // TODO: Delete (has its own store now)
   /** @deprecated has its own store now */
   balancesMetadata: Array<BalancesMetadata>
+
+  hasCheckMetadataHash?: boolean // Indicates if the chain has the CheckMetadataHash extension, enabling signing with ledger generic app
```
Comment (chidg) [inline packages/chaindata-provider/src/types/Chain.ts:43]: Just curious, how do we identify if a chain has this extension, and do we need to maintain this manually in chaindata?

Comment (0xKheops) [reply packages/chaindata-provider/src/types/Chain.ts:43]: to know, need to download the metadata and see if there is a the CheckMetadataHash entry in the extrinsic's signedExtensions

It's done here automatically in chaindata :
https://github.com/TalismanSociety/chaindata/blob/feat/hasCheckMetadataHash-flag/scripts/fetch-external/steps/updateChainsExtrasCache.ts#L361

**Review by chidg — APPROVED** [review]
LGTM

**Pull request merged**